.github/workflows/ci.yml

```
@@ -678,9 +674,23 @@ jobs:
         ./util/opensslwrap.sh version -c
     - name: test external oqs-provider
       run: make test TESTS="test_external_oqsprovider"
-    # Disabled temporarily: https://github.com/latchset/pkcs11-provider/pull/525#discussion_r1982805969
-    # - name: test external pkcs11-provider
-    #   run: make test TESTS="test_external_pkcs11_provider" VERBOSE=1
+
+  external-tests-pkcs11-provider:
+    runs-on: ubuntu-latest
+    container: fedora:rawhide
+    steps:
+    - name: package installs
+      run: |
+        dnf install -y kryoptic perl git meson opensc expect
+    - uses: actions/checkout@v4
+      with:
+        submodules: true  
+    - name: config
+      run: ./config --strict-warnings --banner=Configured --debug enable-external-tests && perl configdata.pm --dump
+    - name: make
+      run: make -s -j4
+    - name: test external pkcs11-provider
+      run: make test TESTS="test_external_pkcs11_provider" VERBOSE=1
```

Why this wouldn't work (possibly with adding other -test_external...)?
```suggestion
    - name: remaining tests on Fedora
      run: make test TESTS="-test_external_pkcs11_provider" HARNESS_JOBS=${HARNESS_JOBS:-4}
```

---

If we want a separate Fedora CI test, it should be put in os-zoo.yml as IMO we do not want to add additional CI jobs into on pull request CI.

---

Good point, should be done with [8d4d230](https://github.com/openssl/openssl/pull/27602/commits/8d4d23034059ee505a235775f679b0552daa5cf6) (only I disabled other external tests to make it faster).